content/en/docs/quick-start/all-in-one-on-linux.md

```diff
@@ -37,9 +37,25 @@ The system requirements above and the instructions below are for the default min
 
 - The node can be accessed through `SSH`.
 - `sudo`/`curl`/`openssl` should be used.
-- `ebtables`/`socat`/`ipset`/`conntrack` should be installed in advance.
 - `docker` can be installed by yourself or by KubeKey.
```

Comment:
Should we declare that docker need to be install by themselves in an offline environment.

Reply:
Added in a note to highlight this hint.